style.css

```diff
@@ -269,3 +270,68 @@ html {
   border-bottom: 2px solid #dfe1e6;
   margin-bottom: 23px;
 }
+
+/* CONTACT ME SECTION */
+
+#contact-me {
+  background-color: #6070ff;
+  border-top-left-radius: 100px;
+  display: flex;
+  flex-direction: column;
+  background-image: url(Images_Background/contact\ form\ shapes-mobile.svg);
+  background-size: 60%;
+  background-position-y: 21px;
+  background-position-x: right;
+  background-repeat: no-repeat;
+  position: relative;
+}
+
+.contact-me-title {
+  font-size: 2.5rem;
+  font-weight: 700;
+  color: #fff;
+  margin-bottom: 12px;
+  text-align: center;
+  margin-top: 25%;
+  flex-shrink: 3;
+}
+
+.contact-me-paragraph {
+  font-weight: 400;
+  font-size: 1.25rem;
+  color: #ebebff;
+  margin-bottom: 52px;
+}
+
+.form-input {
+  color: #172b4d;
+  font-size: 1.0625rem;
+  font-weight: 400;
+}
+
+.form-input-width {
+  width: 100%;
+  margin-bottom: 18px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  outline: none;
+  padding: 15px 16px;
+}
+
+.textarea-size {
+  height: 9.375rem;
+}
+
```

Comment:
- [ ] I noticed that the recommended `font-family` is not applied on the `textarea's placeholder`. You can [check out this link](https://redstapler.co/how-to-change-placeholder-font-css/) to learn how you can do that 💪🛠 Similarly, please use the given `color` as well. You can learn about changing placeholder's color from [here](https://www.w3schools.com/howto/howto_css_placeholder.asp) ℹ

Your design | Figma design 
---| ---| 
![image](https://user-images.githubusercontent.com/106119105/235757789-544ce0c4-f8f0-42d7-8c52-6f8ae9f5ea64.png) | ![image](https://user-images.githubusercontent.com/106119105/235757723-b5b32023-34fe-4ef6-9a66-b37871b4201c.png)